bin/rails

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env ruby
 APP_PATH = File.expand_path('../config/application', __dir__)
-require_relative '../config/boot'
-require 'rails/commands'
+require_relative "../config/boot"
+require "rails/commands"
```

Comment:
Please set IDE to use single quote unless the double quote is really needed for string interpolation.